zerver/webhooks/semaphore/doc.md

```diff
@@ -1,22 +1,36 @@
+# Zulip Semaphore integration
+
 Get Zulip notifications for your Semaphore builds!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. In Semaphore 2.0, under **Configuration** select **Notifications**. Click on
-   **Create New Notification**. Add the URL constructed above to the Webhook
-   **Endpoint** field.
+    **Create New Notification**. Add the URL generated above to the Webhook
+    **Endpoint** field.
+
+    !!! warn ""
```

Comment:
I'm not sure this would count as a warning since it's an alternative step. But, I'm not sure about the right way to format this either.

Reply:
Removed as Semaphore Classic reached 'end of life' in 2023.